magenta/music/musicxml_parser.py

```diff
@@ -374,8 +379,13 @@ def __init__(self, xml_measure, state):
     self.tempos = []
     self.time_signature = None
```

Comment:
That's a very complicated question. Here are some potential answers:
- According to the MusicXML DTDs, it is possible for a time signature to change at any time within a measure. In common practice notation, this is extremely unusual. A time signature only occurs at the beginning of a measure, and a new time signature indicates the start of a new measure, with the exception of a courtesy time signature in a composite time signature (see below).
- It is possible to have a composite time signature such as 2/4 + 3/8 at the beginning of a measure. My code currently does not support this. 
- It is possible to have alternating time signatures, both defined at the beginning of a measure, such as 3/4 6/8. My code currently does not support this. In both this and the previous case, my code will currently only pick up the last time signature.
- It is also possible to have different time signatures within the same measure across different parts (polymeter). Since no major notation program currently supports this, I am not supporting it either.

Do you have any suggestions on how to proceed? All of these cases are edge cases; it is generally safe to assume one time signature per measure in 95%+ of music. I'm not even sure how notation software would encode this because engravers often have to use workarounds to get these edge cases to display, so the output might not be reliable or consistent.